src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/nopol/AbstractNopolRepair.java

```diff
@@ -134,7 +135,7 @@ protected void runNopol(Set<FailureLocation> failureLocation, List<String> tests
         nopolContext.setMaxTimeInMinutes(timeout);
         nopolContext.setLocalizer(NopolContext.NopolLocalizer.COCOSPOON);
         nopolContext.setSolverPath(this.getConfig().getZ3solverPath());
-        nopolContext.setSynthesis(NopolContext.NopolSynthesis.DYNAMOTH);
```

Comment:
could you add a comment to explain why we cannot use DYNAMOTH?

Reply:
Done!